nondex-test/pom.xml

```diff
@@ -108,5 +108,12 @@
       <artifactId>nondex-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
+      <version>4.1</version>
+      <scope>system</scope>
+      <systemPath>/Users/kaku/Desktop/commons-collections4-4.1.jar</systemPath>
```

Comment:
do not add this line here. This causes maven to not download the dependency into your local mvn repo.